pkgs/development/compilers/ghc/9.2.4.nix

```diff
@@ -194,6 +194,13 @@ stdenv.mkDerivation (rec {
       extraPrefix = "utils/haddock/";
       stripLen = 1;
     })
+    (fetchpatch {
+      url = "https://github.com/haskell/cabal/commit/6c796218c92f93c95e94d5ec2d077f6956f68e98.patch";
+      sha256 = "sha256-yRQ6YmMiwBwiYseC5BsrEtDgFbWvst+maGgDtdD0vAY=";
+
+      stripLen = 1;
+      extraPrefix = "libraries/Cabal/";
+    })
```

Comment:
Could you add a comment about what this patch is for, when it is needed, when you expect we'd be able to remove it, etc?

Reply:
Comment added. Let me know if it needs more detail.